OpenRA.Mods.Common/Widgets/Logic/Ingame/ObserverStatsLogic.cs

```diff
@@ -272,9 +272,9 @@ void DisplayStats(Func<Player, ScrollItemWidget> createItem, ModData modData)
 					tt.IgnoreMouseOver = true;
 
 					var teamLabel = tt.Get<LabelWidget>("TEAM");
-					var teamText = modData.Translation.GetString(Team, Translation.Arguments("team-no-team", team.Key));
+					var teamText = modData.Translation.GetString(Team, Translation.Arguments("team", team.Key));
```

Comment:
`team-no-team` is correct.

<img width="702" alt="Screenshot 2022-08-29 at 14 30 37" src="https://user-images.githubusercontent.com/37534529/187191609-fc17bb88-21e8-4193-9a8d-d6f227d4bdb6.png">

Reply:
The parameter is `$team` here, that is the bug!

Reply:
Notice that the translation *key* passed to `GetString` is `Team` which resolves to `team-no-team`.

Reply:
ah right